conferences/SC21/bof_draft.md

```diff
@@ -0,0 +1,115 @@
+
+### Who we are
+
+The [HPC Carpentry][1] community, a volunteer group of high-performance
+computing practitioners lead by a [steering committee][6], has developed a
+number of introductory lessons for new users of HPC clusters.
+
+### What Problem We Solve
+
+It is almost a cliche at this point that greater and greater computational
+capabilities are becoming increasingly affordable. An under-appreciated
+consequence of this is that the range of backgrounds of new users of these
+resources is increasingly broad, and may not include knowledge of computer
+system configurations and file system layouts. There is an evident on-going
+need to address the gap between the needs of these users and the capabilities
+and opportunities presented by the available HPC resources.
+
+The [HPC Carpentry][1] lessons address this gap, building on the pedagogically
+sound instructional methods of the larger [Carpentries][2] community to develop
+several lessons. The best-developed of these, the [HPC Intro][3] lesson, was
+recently submitted to the [Carpentries Incubator][7] program, and is on track
+to become an official [Carpentries][2] lesson. Other important lessons in
+development include a [parallel python][4] lesson, and a lesson focusing on the
+[chapel][5] language.
+
+### Goals for the BoF
+
+During this BoF, we will gather community feedback on our progress to date, to
+ensure that we are aligned with current HPC best practices, and to help the HPC
+Carpentries development team prioritize efforts for successor lessons in light
+of the changing landscape of current HPC practice.
+
+MPI remains a very popular parallel framework, but the community of novice HPC
+users may be using codes for which the MPI details are hidden, or which use
+entirely different parallel frameworks. What is the right level of detail to
+address parallel frameworks in the introductory and advanced levels?
+
+There are many languages in which parallel and HPC codes are written, including
+legacy Fortran codes, low-level C and C++ codes, and newer HPC codes written in
+Python, Julia, or other parallel languages. How can we provide relevant
+information across these communities, while retaining pedagogical rigor?
+
+## Interactivity
+
+Approximately 2/3 to 3/4 of the session will be interactive.
+
+## Format of the non-audience-participation portion
+
+Presentations by the session leaders, either with slides or walking through a
+lesson.
+
+## Commercial?
+
+No.
+
+## Format
+
+The BoF will open with a presentation of the current state of the lessons,
+including "pain points" identified by the development team, after which an open
+discussion will be held to solicit feedback on thees points, and the questions
+above, from the attendees.
+
+## Prior BoFs?
+
+An HPC Carpentry BoF was held at SC17, and an informal group of about fifteen
+participants got together at SC18.
+
+## Time, expected attendance, keywords
+
+ - We expect the session will last 2.5 to 3 hours.
```

Comment:
According to the SC schedule ([here](https://sc21.supercomputing.org/submit/birds-feather-submissions/#section10)), you only get either 1 or 1.5 hours